src/g_text.c

```diff
@@ -237,7 +237,6 @@ void canvas_iemguis(t_glist *gl, t_symbol *guiobjname)
 
 
     pd_vmess(&gl->gl_pd, gensym("editmode"), "i", 1);
- //   glist_noselect(gl);
```

Comment:
oh, i asked "why?" because i didn't understand why you disabled the call to `glist_noselect()`, rather than why you didn't just remove the entire line. I agree that it should be removed rather than just commented out; but the question remains: why it you don't want to call `glist_noselect()`

Reply:
Because I couldn't figure the effect of this call,  because it is not called in `canvas_obj()`, and overall because `glist_noselect()` is already called inside `canvas_howputnew()` ;-)